moskito-frontend/src/Pages/Landing.js

```diff
@@ -0,0 +1,34 @@
+
+import { Link } from 'react-router-dom'
+import Logo from "../Components/Logo/Logo";
+import styled from 'styled-components/macro'
+import FrontHeadline from '../Components/FrontHeadline/FrontHeadline'
+import Button from '../Components/Button/Button';
+
+
+export default function Landing() {
+
+    return (
+        <div>
```

Comment:
As you don't use the `div` for styling a React fragment is sufficient here, no need for the `div`

```suggestion
        <>
```